### DISCUSSION TITLE
Δηλώσεις για τις παρουσιάσεις ερευνητικών άρθρων ΓΕ1 Μέρος Δ

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#21] [Show and tell]
Στα παρακάτω σχόλια μπορείτε να κάνετε δήλωση πρόθεσης της προαιρετικής παρουσίασης ερευνητικού άρθρου με την ανάρτηση  τουλάχιστον μια μέρα πριν την προθεσμία, δλδ **Παρασκευή 28 Οκτωβρίου** των παρακάτω στοιχείων:

1. της βιβλιογραφικής αναφοράς του άρθρου
2. συνδέσμου προς το άρθρο
3. του τελικού αρχείου παρουσίασης

Οι παρουσιάσεις θα γίνουν με σειρά προτεραιότητας ανάλογα με τον χρόνο δήλωσης παρακάτω.

Σε περίπτωση που ο χρόνος της ΟΣΣ1 δεν είναι αρκετός τότε κάποιες παρουσιάσεις μπορεί να γίνουν σε επόμενη ΟΣΣ

**Comment by sotkot** [comment]
Καλησπέρα σας,

Προτίθεμαι να παρουσιάσω για το Μέρος Δ της εργασίας. Ακολουθούν τα σχετικά στοιχεία:

1. Christian Holz and Edward J. Wang. 2017. Glabella: Continuously Sensing Blood Pressure Behavior using an Unobtrusive
Wearable Device. Proc. ACM Interact. Mob. Wearable Ubiquitous Technol. 1, 3, Article 58 (September 2017), 23 pages.

2. DOI: http://doi.org/10.1145/3132024

3. [SDY - GE1 - Part D - Kotrotsios Sotirios.pdf](https://github.com/sdy60/syllabus/files/9872127/SDY.-.GE1.-.Part.D.-.Kotrotsios.Sotirios.pdf)

**Comment by afoscolou** [comment]
Θα ήθελα να κάνω δήλωση πρόθεσης της προαιρετικής παρουσίασης ενός ερευνητικού άρθρου για το Δ' μέρος της 1ης εργασίας.

Παραθέτω τα ζητούμενα:

1.   Young-Ho Kim, Jae Ho Jeon, Bongshin Lee, Eun Kyoung Choe, and Jinwook Seo. 2017. OmniTrack: A Flexible Self-Tracking
Approach Leveraging Semi-Automated Tracking. Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technologies, Vol. 1, No. 3, Article 67.  (September 2017), 28 pages.

2.   https://dl.acm.org/doi/10.1145/3130930

3.   [SDY60_1ERG_PRESENTATION_FOSCOLOU_AGNI.pdf](https://github.com/sdy60/syllabus/files/9872201/SDY60_1ERG_PRESENTATION_FOSCOLOU_AGNI.pdf)

**Comment by ghost** [comment]
Βρίσκομαι στη διάθεσή σας για παρουσίαση ερευνητικού άρθρου.
1. Ludwig, T., Kotthaus, C., Reuter, C., Van Dongen, S., & Pipek, V. (2017). Situated crowdsourcing during disasters: Managing the tasks of spontaneous volunteers through public displays. International Journal of Human-Computer Studies, 102, 103-121.
2. [https://doi.org/10.1016/j.ijhcs.2016.09.008](https://doi.org/10.1016/j.ijhcs.2016.09.008)
3. [Kampouropoulos GE1 PART4.pdf](https://github.com/sdy60/syllabus/files/9891077/Kampouropoulos.GE1.PART4.pdf)

**Comment by gregtsag** [comment]
Βρείτε κάτωθι το άρθρο που στο οποίο θα αναφέρεται η παρουσίαση μου

1. Jiahui Wang, Pavlo Antonenko, Mehmet Celepkolu, Yerika Jimenez, Ethan Fieldman & Ashley Fieldman (2019). Exploring Relationships Between Eye Tracking and Traditional Usability Testing Data. International Journal of Human-Computer Interaction, Volume 35, Issue 6

2, https://doi.org/10.1080/10447318.2018.1464776

3. [ΣΔΥ60_ΓΕ1_ΜΕΡΟΣ_Δ.pptx](https://github.com/sdy60/syllabus/files/9892390/60_.1_._.pptx)
